src/basic/mountpoint-util.c

```diff
@@ -807,6 +807,33 @@ int mount_option_supported(const char *fstype, const char *key, const char *valu
         return true; /* works! */
 }
 
+bool fsconfig_bpffs_supported(void) {
```

Comment:
Then, this can be a static function in core/namespace.c

Reply:
Moved to core/namespace.c, but it is also used in test. So, cannot be static.